src/app/pages/resources/dashboard/dashboard-wrapper.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Dashboard from './dashboard';
+
+import AppWrapper from 'app-wrapper';
+
+function DashboardWrapper() {
```

Comment:
```suggestion
function ChplDashboardWrapper() {
```

Our naming convention is that our components all start with Chpl